### ISSUE TITLE
Recommend settings?

### ISSUE THREAD

**Issue by evancasey** [BisonTrails/lpt-miner#3]
Hey thanks for making this!

Do you have any advice on what values of `MAX_GAS_PRICE` and `MIN_GAS_PRICE` I should use?

**Comment by aaronhenshaw** [comment]
@evancasey MIN_GAS_PRICE is much less important than MAX_GAS_PRICE.

for MAX, you should think about what you want to pay for an LPT, I am currently using 7210000099 which keeps $/LPT under $1 (more like 80 cents I think). That represents a max of 7.2gwei gas price. Sorry for using wei, I know its hard to parse.

The thing MAX protects against is when the FOMO3Ds or baseball cryptos of the world jack the gas up to 20gwei+ you dont wind up paying way too much for LPT.

You can check out https://ethgasstation.info/ for current prices and adjust upwards as you feel its necessary.

**Comment by rstormsf** [comment]
```
general exception Error: could not find key file for address xxx
```
?
I put the file in it, it cannot find it..


## Update
Fixed. I had to same lowercase name for keystore folder without `0x`

**Comment by evancasey** [comment]
@aaronhenshaw awesome, thanks!!
